### DISCUSSION TITLE
Duothan 3.0 - The Buildathon Submissions

### DISCUSSION THREAD

**Discussion by srimalonline** [IEEE-Student-Branch-NSBM/duothan-feb2023-guidelines#1] [Announcements]
🔒 Submission Closed 🔒. Please find the winners & the final score [here](https://github.com/IEEE-Student-Branch-NSBM/duothan-feb2023-guidelines/blob/main/FinalScore.md)👌

## What is this?
This discussion is for submissions to
**Duothan 3.0 - THE LARGEST INTER-UNIVERSITY BUILDATHON IN SRI LANKA!** 😎💕
For more info go to [this website](https://duothan.ieeensbm.org).

### Submission guidelines
To participate in Duothan 3.0, the GitHub repository you submit must follow these guidelines. See [this repo](https://github.com/IEEE-Student-Branch-NSBM/IEEENSBM) for a good example submission.

If you have any questions, feel free to ask from [Organizing Committee](https://www.facebook.com/ieeensbm) 😎❤️

Submission deadline: **Feb 24, 4.30 pm**

- **After 4.30 PM your repository/repositories must be set to Public**
- **Before 4.30 PM your repository/repositories must be set to Private**
- **After 4.30 PM there should not be any commits on your Repositories**

To submit your repo, please make a comment with the following inputs:

- Team ID:
- Team Name:
- Team University:
- Team Members: Names & GitHub Handles 
   mention your team members with their GitHub usernames.
   ex: Srimal Fernando @srimalonline 
        Prineth Fernando @prineth 
        Avin Divakara @AVDiv 
        Senith Edirisingle @senithde-07 
- Repos link: Add Repo links 
- Submission Technology Stacks/Frameworks: Choose tags (ex: `React`  `Node.js`  `MySQL` )
- Work-thru Video link: Add a link (maximum **2 minutes)**

**Comment by mirosh-kavinda** [comment]
Team ID: 20
Team Name: codesouls
Team University: CINEC


Git : [ https://github.com/mirosh-kavinda/medmapweb.git](url)

**Reply by lakminagamage** [reply]
Submission guidelines

To participate in Duothan 3.0, the GitHub repository you submit must follow these guidelines. See [this repo](https://github.com/IEEE-Student-Branch-NSBM/IEEENSBM) for a good example submission.

If you have any questions, feel free to ask from Organizing committee 😎❤️

Submission deadline: Feb 24, 4.30 pm

After 4.30PM your repository/repositories must be set to Public
Before 4.30PM your repository/repositories must be set to Private
After 4.30PM there should not be any commits on your Repositories
To submit your repo, please make a comment with the following inputs:

Team ID: 53
Team Name: Team TCK
Team University: University of Moratuwa
Team Members: Names & GitHub Handles 
mention your team members with their GitHub usernames.
Lakmina Gamage : lakminagamage
Sankha Bimsara : sankhaBA
Vishwani Bhagya : vshwanilgv
Nethmi Nawanjana : NethmiR
Repos link: https://github.com/lakminagamage/duothon-medicineapp.git
Submission Technology Stacks/Frameworks: Express, Node, Bootstrap5
Work-thru Video link: https://drive.google.com/drive/folders/1M92v27Fm_UoZWjLYh2XAaTROtvL2F7tg?usp=sharing

**Reply by thusha88** [reply]
Team ID: 37
Team Name: Debuggers
Team University: SLTC
Team Members: Hasith Malaka - @hmrsl
Randitha Senarathne - @Randitha
Nethmi Rajapaksha - @Nethuu-uni
Thushani Kavindya - @thusha88
Repos link: https://github.com/hmrsl/MedList
Submission Technology Stacks/Frameworks: html/css/java/js/mysql
Work-thru Video link: 
https://drive.google.com/drive/folders/1amIz_E6TM64DYTnM20OusWofVU4ZQm0m?usp=share_link

**Reply by Hashara13** [reply]
38
Kick Bit Fever
Sabaragamuwa University Of Sri Lanka

Hashara Nethmin @Hashara13
Hasinu Ravishka @Hasinu20
Tharani de Silva @Tharaa17
Chemini Nethhari Perera @nethhari

https://github.com/Hashara13/Duothan-3.0.git

HTML CSS JavaScript php MySql

https://drive.google.com/file/d/1LBTKoB0JamGIj8qX7hFdimLV_RjNfLtQ/view?usp=sharing

**Comment by booshananamudara** [comment]
Team ID: 41
Team Name:DeltaX
Team University:University of Moratuwa
Team Members: booshana namudara - @booshananamudara
aruna priyadarshana - @arunapriyadarshana
nawoda thathsarani - @navzt
senuri mallikarachchi - @SenuDyl


Repos link: https://github.com/arunapriyadarshana/medical-web-app.git
Submission Technology Stacks/Frameworks: html/css/bootstrap/php/mysql
Work-thru Video link:

**Reply by PraveenDev111** [reply]
Team ID: 45
Team Name: CookieArmy
Team University: NSBM
Team Members: 
Lakshitha @MKLakshitha
Praveen @PraveenDev111
Sasini @ahsbkdsilva
Umaya @@UmayaWeerasinghe
Repos link: https://github.com/MKLakshitha/Duothan-Cookie-Army-
Submission Technology Stacks/Frameworks: MYSQL, java, jsp, bootstrap, react, js, 
Work-thru Video link: https://drive.google.com/file/d/1yVKh_NirYVpD7rKvgdFG_JiafzynSbR4/view?usp=drivesdk

**Comment by pasinduX** [comment]
IIT BinaryFetch 

Group No = 75

pasindu @pasindu99-ui
maleesha @Maleesha777
indumini @KavyaAmarasena
shehan @ShehanAmantha1

repo link = https://github.com/pasindu99-ui/Duothan-3.0.git

Submission Technology Stacks = html, css,node with express framework, mysql,postman

**Comment by this8** [comment]
Team ID: 58
Team Name: Unstoppable
Team University: NSBM

Team Members: SMTM Samarakoon @this8
WPHK Weerasinghe @wphkweerasinghe
ADK Kaushalya @KushaniKaushalya
CD Egodage @chathura456

Repos link: https://github.com/this8/duothan-unstoppable

Submission Technology Stacks/Frameworks: ASP.NET, SQL Server, Bootstrap

Work-thru Video link: https://github.com/this8/duothan-unstoppable (Video link in Readme file)

**Comment by TeamHelloWorld10** [comment]
1. Team ID: 16
2. Team Name: Team Hello World
3. Team University: University of Moratuwa
4. Team Members: 
Thakshaka Rathnayake - @thakshaka
Kavindu Damsith - @kavindu-damsith
Hasitha Gallella - @HasithaGallella
Lasitha Amarasinghe - @LasithaAmarasinghe

5. Repos link: https://github.com/TeamHelloWorld10/Duothan3.0
6. Submission Technology Stacks/Frameworks: PHP, MySQL, HTML, Bootstrap, JQuery, CSS, JavaScript, Xampp, Postman API
7. Work-thru Video link: https://drive.google.com/drive/folders/154bBNDL7xbsX93hLMTVoR4oqH15ESM0Z?usp=share_link

**Comment by ghost** [comment]
Team ID: 65
Team Name: Ctrl Alt Del
Team University: University Of Moratuwa
Team Members: 
mention your team members with their GitHub usernames.
ex: Malith- Malith21
Pramesh - Fernandoezz
Kavya -  6614Kavya
Sachini - Sahasra0108  
Repos link: https://malith21.github.io/ctrl_alt_del/
Submission Technology Stacks/Frameworks: HTML, CSS, JS
Work-thru Video link: [Add link (maximum 2 minutes)](https://drive.google.com/drive/folders/1R6NzZ4wIX4RcXO1WyUvz3je6wD3MSzWZ?usp=share_link)

**Comment by nushankodikara** [comment]
•	Team ID:40
•	Team Name: hubnerite 
•	Team University: NSBM Green University  
•	Team Members: 
•	Kodikara Nushan Thilanka Prasad Silva   = @nushankodikara 
•	Dasanayaka Mudiyanselage Namodya Chathurani Dasanayaka = @namodyachathurani
•	Y M Divyangi Sansala Yapa = @divsansi
•	S M S Wiratunga = @sandunwira

•	Repos link:  https://github.com/nushankodikara/duothan3.0 
Video : https://www.youtube.com/watch?v=-lb844PGGcg
•	Submission Technology Stacks/Frameworks:
o	MongoDB,
o	Prisma, 
o	Sveltekit, 
o	Node.js,
o	Tailwind

**Comment by Santhusha-bit** [comment]
Team ID: 29
Team Name: Unity
Team University: University of Moratuwa
Team Members: Santhusha Mudannayaka @Santhusha-bit
Sasika Amarasinghe @SasikaA073
Chathura De Silva @Chathura7357
Chamika Jayasinghe @ChamikaJaya
Repos link: https://github.com/SasikaA073/duothon-team
Submission Technology Stacks/Frameworks: Django
Work-thru Video link: https://dms.uom.lk/s/ABgk9CJ4Lw3Fnrr

**Comment by DilukM** [comment]
<h2>MedHub</h2>
<br>Team ID: 05
<br>Team Name: Cyborgs
<br>Team University: CINEC


<b>Team Members:</b>
Diluk Mihiranga <a href="https://github.com/DilukM/">@DilukM</a>
Andrea Okeena <a href="https://github.com/AndyKeena/">@AndyKeena</a>
Udara Madusanka <a href="https://github.com/MADUDARA/">@MADUDARA</a>
Ishani Sewmini <a href="https://github.com/SewminiJ/">@SewminiJ</a>

<b>Technology</b>
HTML
CSS
JAVASCRIPT
Firebase

Repo link: https://github.com/DilukM/MedHub.git

Site link: https://med-hub.netlify.app

**Comment by OshadiSavidya** [comment]
Team ID: 27
Team Name: InVictus
Team University: NSBM

Team Members: WAMD Wiyathunga @MokshaDill
TRS Rathnayaka @trsrathnayaka
KAO Savidya @OshadiSavidya
THCK Thambugala @chathurith

Repos link: https://github.com/MokshaDill/Duothon3.0

Submission Technology Stacks/Frameworks: JAVA, Servlet, SQL Server, Bootstrap, Html ,css ,javascript, php

Work-thru Video link: https://github.com/MokshaDill/Duothon3.0/blob/main/Video

**Comment by NaviRocker** [comment]
- Team ID: 21
- Team Name: Team X-ite
- Team University: University of Colombo School of Computing
- Team Members:
  Rashmi Gunawardana @RashmiNirasha
  Namadee Shakya @namadee
  Ruchira Bogahawatta @ruchira-bogahawatta
  Naveen Rajan @NaviRocker
- Repos link: https://github.com/NaviRocker/MediFy
- Submission Technology Stacks/Frameworks: HTML, JS, CSS, PHP, MySQL, MVC
- Work-thru Video link: https://drive.google.com/drive/folders/17P4ptngEKMdJITwA41Hrz-6h_lJAgQnk?usp=sharing

**Comment by Asamahamed** [comment]
Team ID: 43
Team Name: The Innovators
Team University: NSBM
Team Members:
mention your team members with their GitHub usernames.
 ASAM- AsamAhamed
km Askar - km11askar 
Arsath-  sajahanArsath
aahasan - abdurrahman9920
Repos link: https://github.com/Asamahamed/Final.git
Submission Technology Stacks/Frameworks: HTML, CSS, JS, Reactjs, SpringBoot
Work-thru Video link: Add link (maximum 2 minutes)

**Comment by NethunRanasinghe** [comment]
Team ID: 72
Team Name: 3SN
Team University: NSBM
Team Members: 
	Nethun Ranasingha - @NethunRanasinghe
	Sanidu Anupama -  @saniduanupama
	Sajeewa Wijesinghe -  @methjaya
	Sahansa Jayawardhana - @xyzrw

Repos link:  https://github.com/NethunRanasinghe/Duothon_3.0_3SN
Submission Technology Stacks/Frameworks: Java, JSP, Servlet,  MySQL, Tailwind
Work-thru Video link :

**Comment by ashandimantha2000** [comment]
Team ID:18
Team Name: Hello World
Team University: NSBM Green University
Team Members:
Ashan Dimantha Herath @ashandimantha2000
Dushan Senadheera @DushanSenadheera
Tharun Thathsara Alvitigala @Tharun-Thathsara
Amith Indika @AmithGamagedara


Repos link: https://github.com/DushanSenadheera/HelloWorld.git
Submission Technology Stacks/Frameworks: Choose tags : `php`, `HTML`, `CSS`, `JavaScript`, `MySQL`
Work-thru Video link:

**Comment by pasindusandamal** [comment]
* Team ID: 31
* Team Name: Green Guardians
* Team University: NSBM Green University

* Team Members:              
* P.K.P.Sandamal @sandamal97            
* S.K.V.Sandaruwan @kavindu-sandaruwan            
* H.P.A.Gunarathna @HasithaPramod            
* W.H.P.Fernando @Hima-K

* Repos link: https://github.com/sandamal97/Duothan-3.0.git

* Submission Technology Stacks = html, CSS, framework -  bootstrap  backend database - MySQL

**Comment by Sandaru-J** [comment]
- Team ID: Number 6
- Team Name: HackStreet
- Team University: NSBM
- Team Members: 
Sandaru Jayathilaka @Sandaru-J
Venushka Dhambaraage @VenushkaD
Binuka Silva @binuka02
Nelka Nuvani @nelkanuvani
- Repos link: https://github.com/Sandaru-J/Duothan3.0_IEEE_2023/tree/MEAN_Stack
- Submission Technology Stacks/Frameworks: Choose tags MEAN Stack,FireBase,JWT Towkens 
- Work-thru Video link: https://drive.google.com/drive/folders/1NpCo4TxX4QLYpnVe98RF6bzldRnb1oUN?usp=sharing

**Comment by SaliyaBandara** [comment]
Team ID: 25
Team Name: Love at First Site
Team University: University of Colombo School of Computing
Team Members: 
Saliya Bandara @SaliyaBandara
Nirmal Savinda @nsavinda
Amandi Udawatta @amandiudawatta-uni
Nethmi Athukorala @nkathukorala

Repos link: https://github.com/SaliyaBandara/duothan-task-web
Submission Technology Stacks/Frameworks: PHP, HTML, CSS, JavaScript, JQUERY
Work-thru Video link: https://drive.google.com/drive/folders/1qNKnemtUKh9lanOVuKksClLxyN2O-rJB?usp=share_link

**Comment by tharindug99** [comment]
Team ID: 34
Team Name: SkyWarriors
Team University: Sabaragamuwa University of Sri Lanka
Team Members: @ashalisanjula - Ashali Samaraweera / @tharindug99 - Tharindu Godage / @YaseeRathnayaka - Yasindu Rathnayake / @KavindhyaNirmani - Kavindhya Nirmani
Repos link: https://github.com/tharindug99/SkyWarriors.git
Submission Technology Stacks/Frameworks: MERN
Work-thru Video link:

**Comment by PasinduHar** [comment]
Team ID: 39
Team Name: Dominators
Team University: NSBM
Team Members:
Pasindu Harshana @pasiyaprogramming
Dilukshi Isanka @dilukshi99

Repo link:   https://github.com/pasiyaprogramming/Dominator-Project.git

Submission Technology Stacks = HTML, CSS, framework - bootstrap backend database - MySQL

**Comment by SShehan716** [comment]
Team ID: 35
Team Name: Team No plan
Team University: Informatics Institute of Technology 
Team Members: 
Sachin Shehan - @SShehan716
Yusuf Kaheel - @yusufkhaleel
Savindu Uthsara Panagoda - @savindu29
Keerthana Giridharini - @KeerthanaGiridharini

Repos link: https://github.com/SShehan716/Duothan-TeamNoPlans.git
Submission Technology Stacks/Frameworks: SpringBoot, Html, CSS, JS
Work-thru Video link:

**Comment by NeranajnaNuwani** [comment]
Team ID: 12
Team Name: Team Devil
Team University: NSBM Green University

Team Members:
Neranjana Nuwani @NeranajnaNuwani
Ravindu Malith @RavinduMalith
Navoda Wahalawaththe @Navoda86
Rushmi Prarthana  @RPrarthana

Repos Link : https://github.com/NeranajnaNuwani/Duothon3.0-Final.git
Video link: https://drive.google.com/file/d/1hFIJSaLgBlmNM_t7ZlvEOuB0T-sGv9po/view?usp=sharing
Submission Technologies: HTML, CSS, PHP, MYSQL, Java Script

**Comment by ShamilaWije** [comment]
Team ID: 51
Team Name: the_Buggers
Team University: SLTC
Team Members: 
Kasmira Wijayathunga @kasmirawijayathunga
Shamila Wijekoon @ShamilaWije
Thanura Rukshan @thanurarukshan
Ravindu Prabhash @Ravindu-Prabhash
Repos link:
https://github.com/kasmirawijayathunga/nsbm-duothan-frontend
https://github.com/kasmirawijayathunga/nsbm-duothan-backend
Submission Technology Stacks/Frameworks: reactJs/nextJs, nodeJs/expressJs, mySQL database
Work-thru Video link: https://drive.google.com/file/d/1uL6iU9uuCllvoPDoOgThCGkHiNgvQvAu/view?usp=sharing

**Comment by DidulSeneviratne** [comment]
• Team ID:55
• Team Name: Hypertext Assassins
• Team University: IIT
• Team Members:
• Didul Seneviratne = @DidulSeneviratne
• Sasiru Vishmika = @Sasiru382
• Dimuth Karunarathne = @DimuthKarunathne
• Irudaka Iwantha = @IrudakaIvantha

• Repos link: https://github.com/Sasiru382/Duothan-hypertext-Assassins.git
• Submission Technology Stacks/Frameworks:
o SQL,
o PHP,
o JavaScript,
o CSS
o HTML

**Comment by Manithj** [comment]
Duothan_Script_Kiddies
Duothan competition official repository by team Script Kiddies

Team Script_Kiddies (Team 02)


[Manith Marapperuma](https://github.com/Manithj)

[Ramith Gunawardana](https://github.com/Ramith-Gunawardana)

[Dinithi Amarasinghe](https://github.com/DinaKaush)

[Gayashani Divyanjalee](https://github.com/Gayashani00)

Repos link: https://github.com/Manithj/Duothan_Script_Kiddies

Submission Technology Stacks/Frameworks: @BootStrap @MySQL

Work-thru Video link: https://drive.google.com/file/d/1HyWENCjvxN0tPgga2_gYxn5geP_o_WL2/view?usp=sharing

**Comment by Ranujee123** [comment]
Team ID:59
Team Name: The Byte Builders
Team University: University of Colombo school of computing
Team members:
Vidhara Ranujee @Ranujee123
Nipul Yansith @nipulyansith
Shimra Hanan @shimra-hanan
Lahiru Sandaruwan @Lahiru-S9

Repos link: https://github.com/shimra-hanan/The-Byte-Builders
Submission Technology: HTML, Bulma, CSS, phpmyadmin, C#, MySQL

**Comment by hbkysf27** [comment]
Team ID: 4
Team Name: Time Pass
Team University: NSBM
Team Members: Sivakshan, Yusuf, Kaveeshan, Malsha


Repos link: https://github.com/hbkysf27/douthan-fe.git
Submission Technology Stacks/Frameworks: MEAN STACK
Work-thru Video link: (https://youtu.be/9IQrbZnaQ1Q)

**Comment by sasanka003** [comment]
-  Team ID: 50
-  Team Name: Angry Poop
- Team University: University of Colombo School of Computing
- Team Members:
     Sasanka Premarathna @sasanka003
     Buddhika Nipun @Buddhikanip
     Pinsarie Sigera @pinsariesigera
     Pasindi Vindula @pasi2001

-Repo Link :https://github.com/sasanka003/DuothanWebApp-AngryPoop

- Technologies: HTML, CSS, PHP, MYSQL, Java Script

**Comment by cpaboda** [comment]
Team ID: 63
Team Name: The Amigos
Team University: NSBM Green University
Team Members:
@ThenuraDissanayaka
@KavindaKiridana
@cpaboda
@nevindiweerakkodi

https://github.com/ThenuraDissanayake/Amigos.git
Submission Technology Stacks/Frameworks:
html,css,php,mysql,java script

**Comment by Deshaninayanamini** [comment]
Team ID:  74

Team Name: Codex4

Team University: NSBM Green University

Team Members: Isuru Sandruwan
                          Deshani Liyanage
                          Krishantha Pathum
                          Sachini Dilshani

mention your team members with their GitHub usernames.
Isuru Sandruwan-Isurusandaruwan697
Deshani Liyanage-Deshaninayanamini
Krishantha Pathum-Krishanthapathum
Sachini Dilshani-SachiniDilshani7

Repos link: Add Repo links-https://github.com/Isurusandaruwan697/duthan3.O

Submission Technology Stacks/Frameworks: Choose tags (ex: J2EE,html,css,javascript )

Work-thru Video link: Add link (maximum 2 minutes)

**Comment by LakshanKavindu** [comment]
• Team ID:7
• Team Name:In Ctrl
• Team University: UOM
• Team Members:
• Kavindu Lakshan = @LakshanKavindu
• Nipun madhushan= @Nipun2001M
• Pramudi Gamage = @giftpramu
• Pasindi Wijesuriya = @GPYW

• Repos link: https://github.com/LakshanKavindu/testtwo
• Submission Technology Stacks/Frameworks:
o SQL,
o React js
o JavaScript,
o CSS
o HTML
o Python - flask restful api

**Comment by Isuranga-2001** [comment]
Team ID: 76 
Team Name: 
Rails Rulers 
Team University: University of Moratuwa 
Team Members: 
Isuranga Warnasooriya @isuranga-2001
Iman Thiynaga @imanthiyanga
nethma karunathilaka @Nethma01w
Ruwandi Daramapala @ruwandie
Repos link: https://github.com/Isuranga-2001/MedicineManagementSystem
Submission Technology Stacks/Frameworks: ASP.NET, javascript, css, html

**Comment by isira-adithya** [comment]
### Discussed in https://github.com/IEEE-Student-Branch-NSBM/duothan-feb2023-guidelines/discussions/1

<div type='discussions-op-text'>

<sup>Originally posted by **srimalonline** February 23, 2023</sup>
### Submission guidelines
To participate in Duothan 3.0, the GitHub repository you submit must follow these guidelines. See [this repo](https://github.com/IEEE-Student-Branch-NSBM/IEEENSBM) for a good example submission.

If you have any questions, feel free to ask from Organizing committee 😎❤️

Submission deadline: **Feb 24, 4.30 pm**

- **After 4.30PM your repository/repositories must be set to Public**
- **Before 4.30PM your repository/repositories must be set to Private**
- **After 4.30PM there should not be any commits on your Repositories**

To submit your repo, please make a comment with the following inputs:

- Team ID: 28
- Team Name: Naturals
- Team University: NSBM
- Team Members: Isira Adithya, Vinuka Thejana, Upeksha Indeewara, Vehan Hemsara
   mention your team members with their GitHub usernames.
- Repos link: Add Repo links 
- Submission Technology Stacks/Frameworks: Choose tags (ex: `React`  `Node.js`  `MongoDB` )

Web App URL: http://143.42.64.154:3000/

**Comment by SE-LAPS** [comment]
Duothan 3.0 Web Application

Submission guidelines To participate in Duothan 3.0, the GitHub repository you submit must follow these guidelines. See this repo for a good example submission.

If you have any questions, feel free to ask from Organizing committee 😎❤️

Submission deadline: Feb 24, 4.30 pm

After 4.30PM your repository/repositories must be set to Public Before 4.30PM your repository/repositories must be set to Private After 4.30PM there should not be any commits on your Repositories To submit your repo, please make a comment with the following inputs:

Team ID: 52 
Team Name: Ceylon Coders 
Team University: NSBM Green University
 Team Members: Lahiru Senevirathna @SE-LAPS
SBD Liyanage @SenathLiyanage 
Muthumali Perera @muthumaliperera 
KKS Losathi @LosathiKK
Repos link: https://github.com/SE-LAPS/Duothan-3.0-Web-App 
Submission Technology Stacks/Frameworks: java / servlet / jsp
Work-thru Video link: https://youtu.be/9TWenyEtmgg

**Comment by BUDDHINI02** [comment]
Team ID: 08
Team Name: Code_LionS
Team University: Sabaragamuwa university 

Team Members: 
Buddhini Yallarawa - @BUDDHINI02 
Rasika Wedaarachchi - @rasika51
Susnthan Yuskanthan - @yuskanthan 
Hamitha Apsara - @hamithakd99

Repos link: Add Repo links - https://github.com/yuskanthan/MedFinder.git
Submission Technology Stacks/Frameworks: Choose tags
Work-thru Video link: https://drive.google.com/file/d/1zMnnD3V3X4J4bl6fyG6hx9tOflBhLdAA/view?usp=sharing

**Comment by Chamali02** [comment]
### Team ID: 69
### Team Name: Nano Zillas
### Team University: NSBM
### Team Members: 
 - Hiruna Gayashan @Hirunagrad
 - Chamali Ranasinghe @Chamali02
 - Chethaka Lakshitha @ChethakaL
 - Shamil Keheliya @shamilkeheliya


### Repos link: 
 - https://github.com/shamilkeheliya/IEEE-Frontend-Customers
 - https://github.com/shamilkeheliya/IEEE-Frontend-AdminDashboard
 - https://github.com/shamilkeheliya/IEEE-Frontend-Pharmacy
 - https://github.com/shamilkeheliya/IEEE-Backend
 
### Technology Stacks/Frameworks:
 - MERN
 - React
 - Node JS
 - Express JS
 - Mongo DB
 - Next JS
 
 ### Video Link
 - [Customer Web](https://youtu.be/oyUJbVqFSps) 
 - [Pharmacy Dashboard](https://youtu.be/2NyiGPrBzEc) 
 - [Admin Dashboard](https://youtu.be/ffBahDRqw_A)
 - [Database](https://youtu.be/SXvkNpH3wKc)

**Comment by Hashara13** [comment]
> ### Submission guidelines
> 
> To participate in Duothan 3.0, the GitHub repository you submit must follow these guidelines. See [this repo](https://github.com/IEEE-Student-Branch-NSBM/IEEENSBM) for a good example submission.
> 
> If you have any questions, feel free to ask from Organizing committee 😎❤️
> 
> Submission deadline: **Feb 24, 4.30 pm**
> 
>     * **After 4.30PM your repository/repositories must be set to Public**
> 
>     * **Before 4.30PM your repository/repositories must be set to Private**
> 
>     * **After 4.30PM there should not be any commits on your Repositories**
> 
> 
> To submit your repo, please make a comment with the following inputs:
> 
>     * Team ID:
> 
>     * Team Name:
> 
>     * Team University:
> 
>     * Team Members: Names & GitHub Handles
>       mention your team members with their GitHub usernames.
>       ex: Srimal Fernando @srimalonline
>       Prineth Fernando @prineth
>       Avin Divakara @AVDiv
>       Senith Edirisingle @senithde-07
> 
>     * Repos link: Add Repo links
> 
>     * Submission Technology Stacks/Frameworks: Choose tags (ex: `React`  `Node.js`  `MySQL` )
> 
>     * Work-thru Video link: Add link (maximum **2 minutes)**



38
Kick Bit Fever
Sabaragamuwa University Of Sri Lanka

Hashara Nethmin @Hashara13
Hasinu Ravishka @Hasinu20
Tharani de Silva @Tharaa17
Chemini Nethhari Perera @nethhari

https://github.com/Hashara13/Duothan-3.0.git

HTML CSS JavaScript php MySql

https://drive.google.com/file/d/1LBTKoB0JamGIj8qX7hFdimLV_RjNfLtQ/view?usp=sharing

**Comment by dustyrabbit9** [comment]
**Team ID:** 17
**Team Name:** Lost Frequencies
**Team University:** National Institute of Business Management (NIBM)

**Team Members:** 
Navin Thamindu @navintc
Vileka Karunarathne @Vileka-k
Nimeshika Hansani @Galaxy-420
Imran Rasheed @dustyrabbit9

https://github.com/navintc/NSBM_Duothan
#PHP #Laravel #React #HTML #CSS 
Work-thru Video link: https://drive.google.com/drive/folders/1XHFF3FK2Tv5FDVkO6pWjhEZEEmbd7KWE?usp=sharing

**Comment by Sonal000** [comment]
https://github.com/Sonal000/pharmacy.git

Team No. 64
UCSC
Sonal Induwara
Wasfan Waseem
Dewmini Lochana
Venuri Abeysekara

**Comment by HeshanNavindu-7** [comment]
Team ID:22
Team Name:Dream Hackers
Team University:Sabaragamuwa university of srilanka
Team Members: Heshan Navindu
Nipuni Nadeeshani
Nimtharu Jayasekara
Nayani Tharusika
Repos link:  https://github.com/HeshanNavindu-7/Pharmercys-Nsbm

**Comment by Nipun-Milinda** [comment]
Team ID: 61
Team Name: S2Nx
Team University: University of Moratuwa
Team Members: 
1. Nipun Milinda @Nipun-Milinda
2. Himesh Shiwantha @ShiwanthaH
3. Hasini Sachintha @HasiniSachintha
4. Bhashini Mananadee @Bha3hi

Repos link: https://github.com/orgs/S2NX-Duothan/repositories
Submission Technology Stacks/Frameworks:  MERN
Work-thru Video link: https://drive.google.com/drive/folders/1L-S-VS4WOOTSndHvRedXvgG1Omml6C5Z?usp=share_link

**Comment by Rdilshan** [comment]
## Team ID:

 - 49

 ## Team Name:

 - BigX


 ## Team University:

 - Sabaragamuwa University of Sri Lanka
  ## Team Members:

 - Randika Dilshan
 - Tharidu Hashan
 - Ravidu Bimsara
 - Hashini Wickramashinha
  ## mention your team members with their GitHub usernames:

 - Randika Dilshan-Rdilshan
 - Tharidu Hashan-tharindususl
 - Ravidu Bimsara-ravindu2
 - Hashini Wickramashinha-Hashini22

 ## Submission Technology Stacks/Frameworks:

 - Node JS
 - HTML
 - CSS
 - EJS
 - Express JS
 - MonogDb
## 🔗 Links
 - https://github.com/Rdilshan/Duothan-BigX-
## 🔗 Figma Links
 - https://www.figma.com/file/hDgk3VIyUudoVmSJ04JeBo/Untitled?node-id=1%3A2&t=royVM6bLhRNCGEG1-1

**Comment by Chasith-Randima** [comment]
Teamd ID :01
Team Name : Black_Mamba
Team University : NSBM Green University
Teame Memebers : K.G.C.R.Silva = [@Chasith-Randima](https://github.com/Chasith-Randima)
		Hasini Amarasinghe = [@hasiniamare](https://github.com/hasiniamare)
	        Shakib Jasim = [@shakeeb2001](https://github.com/shakeeb2001)
		harshani = [@harshani00](https://github.com/harshani00)

Repo Links : FrontEnd = https://github.com/Chasith-Randima/Duothan_3.0_FrontEnd
	      Backend = https://github.com/Chasith-Randima/Duothon_backend
Submission Technology Stacks/Frameworks: NodeJS,ExpresJS,ReactJS,TailwindCss,MongoDB

video go through link = https://drive.google.com/drive/folders/1N_QTS-w1dePIjgzRpNj10-83j18nCMVh?usp=share_link

**Comment by GayuniBas** [comment]
Submission guidelines
To participate in Duothan 3.0, the GitHub repository you submit must follow these guidelines. See [this repo](https://github.com/IEEE-Student-Branch-NSBM/IEEENSBM) for a good example submission.

If you have any questions, feel free to ask from Organizing committee 😎❤️

Submission deadline: Feb 24, 4.30 pm

After 4.30PM your repository/repositories must be set to Public
Before 4.30PM your repository/repositories must be set to Private
After 4.30PM there should not be any commits on your Repositories
To submit your repo, please make a comment with the following inputs:

Team ID: 70
Team Name:Code_Breaker
Team University: UoM
Team Members: 
R.L.M.Kavinda (https://github.com/kavindarlm)
Gayuni Basnayake (https://github.com/GayuniBas)
WAJ Amasara (https://github.com/WAJAMASARA)
Anuradha G.W.N (https://github.com/Nipuni31)
Repos link: https://github.com/GayuniBas/code_breakers.git
Submission Technology Stacks/Frameworks: Choose tags (ex: React Node.js MySQL )
Work-thru Video link: Add link (maximum 2 minutes)
Figma UI Design Link - https://www.figma.com/file/VemKERhjqBnMBaPArA8AGm/Untitled?node-id=0%3A1&t=AhApzFsgR268bfsS-1
Drive Link - https://drive.google.com/file/d/19eSxoK6wgAs4lzEfs56Abd9y0SZ1nffu/view?usp=sharing

**Comment by harithaw** [comment]
Team ID: 68
Team Name: S2Ny
Team University: University of Moratuwa
Team Members: Names & GitHub Handles
Haritha Weerathunga @harithaw
Dasun Madusanka @dasun-madusanka
Thikshana Ekanayake @ThikshanaLankangana
Shihara Dissanayake @215131E

Repos link: 
FrontEnd: https://github.com/harithaw/react-template
BackEnd: https://github.com/harithaw/healify-api

Submission Technology Stacks/Frameworks: MERN stack (MongoDB, Express, React, Node)
Work-thru Video link: https://drive.google.com/file/d/1sDuNrKLg0VI5mH6vJvravR055DrMUAh2/view?usp=share_link

**Comment by NMihi** [comment]
Team ID :67
Cinec Campus
CINEC_BitLordz
Team members 👍 :

Nirmal Mihiranga
Suwhas Prabashwara
K.D.D.Rajarathna
Teshan Nuwantha

video link: https://drive.google.com/drive/folders/1UC7SRNJTHfinKH6Ryie0dI28CLhA_ehO?usp=share_link
git hub repository:
https://github.com/NMihi/phillseeker-landing.git

Submission technologies used HTML, PHP, JavaScript, CSS, Mysql, FireBase, NodeJs)

**Comment by YathuNK** [comment]
Team ID: 32 
Team Name: Quaxor
Team University: University of Moratuwa

Team Members: Names & GitHub Handles
Pranavan Jegatheeswaran [@jegatheesprana](https://github.com/jegatheesprana)   
Srikumaran Kajanan [@Kajanan98](https://github.com/Kajanan98)   
Thakshyan Thanabalasingam [@Thakshayan](https://github.com/Thakshayan)   
Yathurshan Kalanantharasan [@YathuNK](https://github.com/YathuNK)

Back-end : https://github.com/jegatheesprana/duathon-api
Front-end : https://github.com/jegatheesprana/duathon-ui

Technology Stacks/Frameworks: Node.js, React, MongoDB, Express.js, Material-UI

**Comment by Thush20210156** [comment]
To participate in Duothan 3.0, the GitHub repository you submit must follow these guidelines. See [this repo](https://github.com/IEEE-Student-Branch-NSBM/IEEENSBM) for a good example submission.

If you have any questions, feel free to ask from Organizing committee 😎❤️

Submission deadline: Feb 24, 4.30 pm

    After 4.30PM your repository/repositories must be set to Public
    Before 4.30PM your repository/repositories must be set to Private
    After 4.30PM there should not be any commits on your Repositories

To submit your repo, please make a comment with the following inputs:

    Team ID: 23
    Team Name: CLOUD_KID
    Team University: IIT
    Team Members: Thushini Abeysuriya@Thush20210156
Sulakshana Karunarathne@sulakshana007
Kalindu Yapa@KalinduYapa
Tharusha Perera@TharushaRehan
    mention your team members with their GitHub usernames.
    ex: Srimal Fernando @srimalonline
    Prineth Fernando @prineth
    Avin Divakara @AVDiv
    Senith Edirisingle @senithde-07
    Repos link: [https://github.com/Thush20210156/CLOUDKID_NSBM_BUILDATHONl](url)
    Submission Technology Stacks/Frameworks: Choose tags (ex: React Node.js MySQL )
    Work-thru Video link: Add link (maximum 2 minutes)

**Reply by DumiaSL** [reply]
Team ID: 62
Team Name: Prime Calculas
Team University: Informatics Institute of Technology (IIT)
gitcode- https://github.com/DumiaSL/NSBM-Prime_Calculas_62.git
